tests/mqtt/test_adapter.py

```
@@ -0,0 +1,735 @@
+        self.broker.stop()
+        self.adapter.stop()
+        MQTTBrokerTest.kill_all_test_brokers()
+
+
+class Test_Unsuccessful_Connection_To_Broker(unittest.TestCase):
+
+    def test_error_logged_if_broker_does_not_exist(self):
+        adapter = MQTTClientAdapter(
+            "some_company", "test_car", timeout=0.5, broker_host="localhost", port=1884, event_queue=EventQueue()
+        )
+        with self.assertLogs(logger=_logger._logger, level="ERROR") as cm:
```

Ruff

Local variable `cm` is assigned to but never used

Remove assignment to unused variable `cm`

(F841)